src/main/scala/mesosphere/marathon/core/base/RichRuntime.scala

```diff
@@ -51,5 +51,6 @@ case class RichRuntime(runtime: Runtime) extends StrictLogging {
 
 object RichRuntime {
   val FatalErrorSignal = 137
+  val SuicideExitCode = 42
```

Comment:
Those vals deserve "talking names" and comments explaining their usage. wdyt?

Reply:
Yup see above.